docs/scos/dev/front-end-development/migration-guide-switch-from-yarn-to-npm.md

```diff
@@ -41,7 +41,18 @@ To ensure the CI jobs will run successfully, add the same config part to all 'de
 
 {% endinfo_block %}
 
-2. Run the following command to pull the latest `docker-sdk` version:
+2. The following commands are deprecated and will work correctly only with the previous versions (using Yarn). 
```

Comment:
What do you mean by "the following commands"? Only commands from lines 46 and 47 below?
Remove deprecated `frontend:yves:install-dependencies` and `frontend:zed:install-dependencies` commands.
Replace the `frontend:mp:install-dependencies` command with the next one `frontend:project:install-dependencies`.

Reply:
These commands are deprecated 
```
frontend:yves:install-dependencies
frontend:zed:install-dependencies
frontend:mp:install-dependencies
```

Now we are using only one command to install all dependencies
```
frontend:project:install-dependencies
```